cypress/plugins/index.js

```diff
@@ -39,6 +39,13 @@ module.exports = (on, config) => {
   config.env.projectFolder = process.env.KIT_TEST_DIR || process.cwd()
   config.env.tempFolder = path.join(__dirname, '..', 'temp')
 
+  const pkg = JSON.parse(fs.readFileSync(path.join(config.env.projectFolder, 'package.json'), 'utf8'))
```

Comment:
small things and up for debate, but personally I'd prefer
- plain english variable name like `package` rather than `pkg`
- to break down something like this into more steps for ease of reading. So something like 
```
const packagePath = path.join(config.env.projectFolder, 'package.json')
```

Reply:
Replaced _pkg_ with _packageObject_ as _package_ is a reserved word.